agent/src/agent/api/main.py

```diff
@@ -38,4 +38,4 @@ def teardown_request_func(exception):
 
 @app.route('/version', methods=['GET'])
 def version():
-    return jsonify('Daria Agent version: ' + __version__)
+    return jsonify('Daria Agent version ' + __version__)
```

Comment:
why?
please check if it won't break release workflow step "check version"

Reply:
I just checked how version is displayed in other apps and made the same, I also think that grammatically it's correct not to use a colon